lib/src/screens/send/send_page.dart

```diff
@@ -439,10 +440,17 @@ class SendPage extends BasePage {
                                 }
 
                                 if (state is TransactionCommitted) {
+                                  String alertContent;
+                                  if (solana != null) {
```

Comment:
this will be true for Cake Wallet app in general, not for solana wallets specifically

Reply:
Alright

Streamlined it to depend not he wallet type itself